pkg/cmd/quota/quota_create.go

```diff
@@ -52,21 +52,21 @@ func newCmdQuotaCreate(ctx api.Context) *cobra.Command {
 				rolesRes <- rolesResult{roles, err}
 			}()
 
-			rolesResult := <-rolesRes
-			if rolesResult.err != nil {
-				return rolesResult.err
+			rResult := <-rolesRes
```

Comment:
I don't think this makes this variable name more expressive. I would rather rename the channel to `rolesResCh` or `rolesResChan`.